addons/account/models/account_reconcile_model.py

```diff
@@ -210,7 +210,7 @@ def _apply_in_bank_widget(self, residual_amount_currency, partner, st_line):
             }
 
         if not aml_vals['name']:
-            aml_vals['name'] = st_line.payment_ref
+            aml_vals['name'] = st_line.payment_ref or '/'
```

Comment:
Payment used to be a required field, but it causes an exception if the bank provides an empty field. It is used in other parts of the code assuming to have value.
(leaving it blank caused another exception but I did not track where it was, I can find it if required)

Reply:
aml.name isn't required, so there's no good reason of forcing it to '/'. Whatever the code failing on an empty string, it has to be fixed rather than hidden